card/mod/05_standard/spec/set/type/search_type_spec.rb

```diff
@@ -35,6 +35,41 @@
     expect(pointer_card.content).to eq(%{{"type":"User"}})
   end
 
+  context 'with right plus array' do
+    it 'render the card list and paging correctly' do
+      Card.create! name: 'Samsung'
+      Card.create! name: 'Samsung+tag'
+      Card.create! name: 'Samsung+source'
+      Card.create! name: 'Apple'
+      Card.create! name: 'Apple+tag'
+      Card.create! name: 'Apple+source'
+      Card.create! name: 'HTC'
+      Card.create! name: 'HTC+tag'
+      Card.create! name: 'HTC+source'
+      search_card = Card.create!(
+        name: 'search_with_right_plus',
+        type_id: Card::SearchTypeID,
+        content: %{
+            {
+              "right_plus":{
+                "name":["in","tag","source"]
+              },
+              "limit":1
+            }
+        }
+      )
+      html = search_card.format.render_open
+      expect(html).to have_tag('ul', with: { class: 'pagination paging' }) do
+        with_tag 'a', text: '1'
+        with_tag 'a', text: '2'
+        with_tag 'a', text: '3'
+      end
+      expect(html).to have_tag('div', with: { class: 'search-result-list' }) do
+        with_tag 'div', with: { id: 'Apple' }
+      end
```

Comment:
This should probably be in Query spec, not in search type spec;  I don't think we need to render any html to test.  Something like:
````
query = { right_plus: ....
Card.search(query).items.should ==
````